src/mds/MDSRank.cc

```diff
@@ -1758,6 +1758,13 @@ bool MDSRankDispatcher::handle_asok_command(
     } else {
       mdcache->dump_cache(path);
     }
+  } else if (command == "dump below") {
+    string root;
+    int64_t depth;
+    cmd_getval(g_ceph_context, cmdmap, "root", root);
+    if (!cmd_getval(g_ceph_context, cmdmap, "depth", depth))
+      depth = -1;
+    mdcache->dump_cache(root, depth, f);
```

Comment:
Why, to avoid changes to files under the target path? Just racing here might be an appropriate choice since this is a debug dump.